### DISCUSSION TITLE
Why not add a path correction module？

### DISCUSSION THREAD

**Discussion by RoboticsPolyu** [ros-industrial/abb_libegm#137] [Q&A] [answered]
EGM have a additional function as Path Correction, which can be use to correction original abb robot motion instruction. Your all_libegm, I did not see this related method.

**Comment by gavanderhoorn** [comment, marked as answer]
The simple answer is: because the developers working on `abb_libegm` didn't have a use for one.

**Comment by gavanderhoorn** [comment]
See #20 for an earlier discussion about this btw.
